docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md

```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
 keywords: time tracking, toggl integration
 ---
 
-# Time tracking with Toggl
+# Time tracking with Toggl Track
 
-To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl (https://toggl.com/).
+To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl Track ([https://toggl.com/track/](https://toggl.com/track/)).
 
-## OpenProject Toggl integration
+## Installation
 
-You can install a browser add-in to track time via Toggl in OpenProject. This way, you can record spent times directly to a work package in OpenProject via a start/stop button. 
+You can install a browser add-in to track time via Toggl Track in OpenProject. This way, you can record spent times directly to a work package in OpenProject via a start/stop button. You need to register an account at Toggle in order to use this feature.
 
-If this add-in is installed, a start/stop button is displayed on the work package details view, which can be used to record times from OpenProject into Toggl:
+This integration can be added for example for Google Chrome as [Toggle Track extension](https://chrome.google.com/webstore/detail/toggl-button-productivity/oejgccbfbmkkpaidnkphaiaecficdnfn) and it also exists for Firefox.
 
-![Toggl-integration](Toggl-integration.png)
+## Configuration
 
-This integration can be added for example for Google Chrome as [Chrome Extension](https://chrome.google.com/webstore/detail/toggl-button-productivity/oejgccbfbmkkpaidnkphaiaecficdnfn).
+On the cogwheel of the toggle track extension you reach the **Settings**, from there chose **Integrations**.
 
-You can then transfer the recorded times from Toggl to OpenProject via the API if needed.
+For OpenProject Domains that are located on openproject.com you are able to chose **Openproject - openroject.com** from the list of default integrations.
+
+For any other OpenProject Domains you need to exactly add the **FQDN** to the **Custom URLs for Integration** and chose **OpenProject**. For example: community.openproject.org (wildcards are not supported by Toggle Track at the moment of writing)
```

Comment:
I believe that the abbreviation FQDM is not a term that most people know. I learnt it just some weeks ago.

```suggestion
For any other OpenProject domains you need to exactly add the exact domain (including the subdomain if present) to the **Custom URLs for Integration** and chose **OpenProject**. For example: community.openproject.org (wildcards are not supported by Toggle Track at the moment of writing)
```